routes/notif_nodemailer.js

```diff
@@ -80,11 +80,12 @@ module.exports = {
     },
 
     sendUser: function(mailOptions, callback) {
+        let sbjtag = (CONFIG.nodemailer.prefix ? CONFIG.nodemailer.prefix : CONFIG.general.name);
 
         let info =  transporter.sendMail({
             from: mailOptions.origin,
             to: mailOptions.destinations.join(),
-            subject: mailOptions.subject,
+            subject: "[" + sbjtag + "] " + mailOptions.subject,
```

Comment:
If sbjtag is left empty you should not writz the brackets

Reply:
https://github.com/genouest/genouestaccountmanager/pull/76/commits/9b711dc89c98904aaba1efcc71eaff85388cf576